domapping/templating.py

```diff
@@ -32,18 +32,19 @@
 from .mapping import clean_mapping
 
 
-def mapping_to_jinja(es_mapping, type_name, indent=2):
+def mapping_to_jinja(es_mapping, type_name, indent=2, start_indent=''):
     """Pretty pring an elasticsearch type mapping as a jinja template.
 
     :param es_mapping: elasticsearch mapping
     :param type_name: name of the elasticsearch type
     :param indent: intentation step
+    :param start_indent: base indent
     """
     result = []
-    result += '{{\n'.format(type=type_name)
+    result += start_indent + '{{\n'.format(type=type_name)
```

Comment:
My bad, I know that I am the one who suggested to add the `start_indent` here. As the calling CLI function starts with
`mykeytype: ` it would not make sense to append it on this line. However the other lines still make sense.